lib/membrane_rtc_engine/endpoints/webrtc_endpoint.ex

```diff
@@ -455,13 +448,14 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC do
   end
 
   @impl true
-  def handle_pad_added(Pad.ref(:input, track_id) = pad, _ctx, state) do
+  def handle_pad_added(Pad.ref(:input, track_id) = pad, ctx, state) do
+    {:endpoint, endpoint_id} = ctx.name
     track = Map.fetch!(state.outbound_tracks, track_id)
     initial_target_variant = state.simulcast_config.initial_target_variant.(track)
 
     links = [
       link_bin_input(pad)
-      |> to({:track_receiver, track_id}, %TrackReceiver{
+      |> to({:track_receiver, endpoint_id, track_id}, %TrackReceiver{
```

Comment:
Why do we need to identify `TrackReceiver` by the id of his parent?

Reply:
Well, the problem is that we ends up with multiple track_receivers with the same id as `track_id` is the id of track that we are receiving not sending :thinking:

Reply:
But `endpoint_id` will also be constant inside WebRTCEndpoint, if I am thinking correctly. So I don't know how it solves a problem.

Reply:
Shouldn't we remove this `endpoint_id` as we added `log_metadata`?